utilities/parsing.py

```diff
@@ -371,8 +371,12 @@ def __call__(self, parser, namespace, values, option_string=None):
             "2016PreVFP",
             "2016PostVFP",
             "2017",
-            "2018",
             "2017H",
+            "2018",
+            "2018A",
```

Comment:
I don't think it is a good idea to define the A,B,C,D here separately. Instead I suggest to define separate keys in dataDictV9_2018 in wremnants/datasets/datasetDict2018_v9.py and then exclude certain samples using the --excludeProcs argument.